etl/steps/data/garden/harvard/2023-09-18/colonial_dates_dataset.py

```diff
@@ -88,10 +92,21 @@ def process_data(tb: Table) -> Table:
     tb_rest = tb_rest.drop(columns=["colstart_max", "colend_max", "colstart_mean", "colend_mean", "col"])
 
     # Create another table with the total number of colonies per colonizer and year
-    tb_count = tb_colonized.groupby(["colonizer", "year"]).agg({"country": "count"}).reset_index().copy_metadata(tb)
+    tb_count = tb_colonized.copy()
+    tb_count = geo.harmonize_countries(df=tb_count, countries_file=paths.country_mapping_path)
+    tb_count = tb_count.merge(tb_pop[["country", "year", "population"]], how="left", on=["country", "year"])
+
+    tb_count = (
+        tb_count.groupby(["colonizer", "year"])
+        .agg({"country": "count", "population": "sum"})
+        .reset_index()
+        .copy_metadata(tb)
+    )
 
     # Rename columns
-    tb_count = tb_count.rename(columns={"colonizer": "country", "country": "total_colonies"})
+    tb_count = tb_count.rename(
+        columns={"colonizer": "country", "country": "total_colonies", "population": "total_colonies_pop"}
```

Comment:
Hi @Marigold! How can avoid the `Table colonial_dates_dataset, column total_colonies_pop has no origins.` message? I assumed it's because it's the population variable of the population dataset, but I don't know how to solve it for it to function properly.

Reply:
I saw you're away. No problem. Could you help me understanding this, @pablorosado?

Reply:
I see you're away, no worries. Could you help me understanding this, @pabloarosado?